libs/ngx-mime/src/lib/core/alto-service/html.formatter.ts

```diff
@@ -34,13 +35,38 @@ export class HtmlFormatter {
         styles.push('font-weight: bold');
       }
 
+      var paragraphs = words.join(' ');
+
+      if(this.searchQuery || this.hits){
+        paragraphs = this.transform(paragraphs, this.searchQuery);
+      }
+
       html += '<p';
       if (styles && styles.length > 0) {
         html += ` style="${styles.join(';')}"`;
       }
-      html += `>${words.join(' ')}<p/>`;
+      html += `>${paragraphs}<p/>`;
     });
 
+
     return this.sanitizer.bypassSecurityTrustHtml(html);
   }
+
+  transform(html: any, searchQuery: any): any {
+    if(this.hits){
+      if(this.hits.length === 1){
+        return this.markMatch(html, this.hits[0].match.trim());
+      } else if (this.hits.length === 0)
+      return html;
+    }
+    for(const text of searchQuery) {
+      html = text.length === 1 ? this.markMatch(html, `(${text}\\.)|( ${text} )`) : this.markMatch(html, "\\b"+text.replace(/[^\w+\søæå]/gi, '')+"\\b");
+    }
+    return html;
+  }
+
+  markMatch(html: any, pat: string){
```

Comment:
Rename parameter to `pattern` for better readability.
```suggestion
  markMatch(html: any, pattern: string){
```